### DISCUSSION TITLE
Any ideas on where to buy ?

### DISCUSSION THREAD

**Discussion by wimg** [jomjol/AI-on-the-edge-device#1684] [Q&A]
Does anyone know where to buy a device that will actually work ?
I tried 6 different sources on Amazon and 2 outside of Amazon. Either they had the wrong chip or only 2MByte of PSRAM :-(

**Comment by jurasjo** [comment]
I have bought 2 pieces here. They are what they claim.
https://www.partco.fi/fi/kehityskortit/esp/24113-esp32-esp32cam.html

It is a Finnish  shop but they send also abroad. Change Suomi to English in the menu.

**Reply by caco3** [reply]
Those work for me:
https://de.aliexpress.com/item/1005004652465892.html?spm=a2g0o.store_pc_allProduct.8148356.2.12e84f6cLfzTEM&pdp_npi=2%40dis%21CHF%21CHF%204.95%21CHF%204.31%21%21%21%21%21%402103250d16678144344273617e15b4%2112000029988384077%21sh

mabe order more than one, on one of mine the cam was bad
